.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "LSP Test - Debug active test",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/bazel-bin/test/lsp_test_runner",
+            "args": ["--single_test=${fileDirname}/${fileBasenameNoExtension}", "--dt-no-colors"],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "lldb",
```

Comment:
🤔  this isn't the third-party VS Code LLDB extension, looks like it's the official VS Code LLDB support discussed here:
https://code.visualstudio.com/docs/cpp/launch-json-reference

Does this require the following not-recommended extension (I hope not)?
https://github.com/sorbet/sorbet/blob/master/.vscode/extensions.json#L8

Reply:
As this been revisited since that was put in place in 2019? I have been using the MS C++ support without issue as of now (no CPU issue at rest from what I can tell)